paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -131,7 +131,8 @@ CpuPassStrategy::CpuPassStrategy() : PassStrategy({}) {
       "conv_eltwiseadd_bn_fuse_pass",  //
       "is_test_pass",                  //
       "identity_scale_op_clean_pass",  //
-      "runtime_context_cache_pass",    //
+      // TODO(?): fix the pass below as it breaks accuracy
+      // "runtime_context_cache_pass",
```

Comment:
Could you not delete `runtime_context_cache_pass` here? I will try to fix this problem in another PR. And your PR could be merged at first.

Reply:
Ah, OK.
Done, both.